src/ontology/config/icd11foundation-property-map.sssom.tsv

```diff
@@ -0,0 +1,4 @@
+subject_id	object_id
+http://purl.org/dc/elements/1.1/source	oboInOwl:source
+http://id.who.int/icd/schema/isObsolote    owl:deprecated
+http://id.who.int/icd/schema/note    rdfs:comment
```

Comment:
## Added `icd11foundation-property-map.sssom.tsv`
**Filename**: The only 2 other such files are called `property-map-1.sssom.tsv` and `property-map-2.sssom.tsv`. But I felt that since this is only being used for 1 source, it would be good to prepend the source name.

**Context**: property mappings decision thread